client/client.go

```diff
@@ -265,30 +291,15 @@ func FakeDoRetry(retry, timeout time.Duration) (restore func()) {
 	}
 }
 
-type hijacked struct {
-	do func(*http.Request) (*http.Response, error)
-}
-
-func (h hijacked) Do(req *http.Request) (*http.Response, error) {
-	return h.do(req)
-}
-
-// Hijack lets the caller take over the raw HTTP request.
-func (client *Client) Hijack(f func(*http.Request) (*http.Response, error)) {
```

Comment:
I think it's worth mentioning in the PR description that we've quietly removed this exported method (and why that's almost certainly okay).

Reply:
I've added this to the PR description, which I will use as the commit message. Resolving.